CGPs/cgp-0145.md

```diff
@@ -0,0 +1,74 @@
+---
+cgp: 154
+title: Increase minimum gas threshold to 25000000000 gwei
+date-created: 2024-08-6
+author: Martín Volpe (@martinvol)
+status: DRAFT
+discussions-to:
```

Comment:
We need a link where to point the discussions.

Reply:
But I can't make the discussion before I have a link to the proposal

Reply:
Normally, people post  in the forum to get feedback on the proposal and after that they submitted the PR linking the PR to the forum discussion.  But I will approve this PR.